README.md

```diff
@@ -1,15 +1,14 @@
 # gmaven_rules
 
-This project generates `gmaven.bzl`, a file containing external repository
-targets for all artifacts in
-[https://maven.google.com](https://maven.google.com).
+This project provides support for easily depening on common Android libraries in Bazel.
 
-# Support Policy
+It provides `gmaven.bzl`, a file containing external repository
+targets for all artifacts in [https://maven.google.com](https://maven.google.com) plus their
```

Comment:
all artifacts on the [Google Maven Repository](https://maven.google.com)

Reply:
Done